cirq/google/__init__.py

```diff
@@ -32,12 +32,12 @@
     PhasedFSimEngineSimulator,
     SQRT_ISWAP_PARAMETERS,
     THETA_ZETA_GAMMA_FLOQUET_PHASED_FSIM_CHARACTERIZATION,
-    make_floquet_request_for_circuit,
-    make_floquet_request_for_moment,
-    zeta_chi_gamma_calibration_for_moments,
-    run_characterizations,
+    prepare_floquet_characterization_for_circuit,
+    prepare_floquet_characterization_for_moment,
+    zeta_chi_gamma_compensation_for_moments,
```

Comment:
everything else has a verb

Reply:
That is true, I think make is the most appropriate (add could also work but only for this particular method, in future there might be more substantial circuit modifications).